docs_headless/src/content/docs/LockOnMount.md

```diff
@@ -0,0 +1,39 @@
+---
+title: '<LockOnMount>'
+---
+
+`<LockOnMount />` calls `dataProvider.lock()` on mount and `dataProvider.unlock()` on unmount to lock and unlock the record. It relies on `authProvider.getIdentity()` to get the identity of the current user. It guesses the current `resource` and `recordId` from the context (or the route) if not provided.
```

Comment:
We should probably add links here:
```suggestion
`<LockOnMount />` calls `dataProvider.lock()` on mount and `dataProvider.unlock()` on unmount to lock and unlock the record. It relies on [`authProvider.getIdentity()`](./AuthProviderWriting.md#getidentity) to get the identity of the current user. It guesses the current `resource` and `recordId` from the context (or the route) if not provided.
```

We really miss an introduction for realtime too, like what we have for Soft Delete, which would allow to link to the dataProvider setup and methods